Firmware/ChameleonMini/Application/MifareUltralight.c

```diff
@@ -467,6 +467,7 @@ uint16_t MifareUltralightAppProcess(uint8_t* Buffer, uint16_t BitCount)
         break;
 
     case STATE_READY1:
+        ByteCount = (BitCount + 7) >> 3;
```

Comment:
Can't these offsets be set as defined macros to ease code reading?

Reply:
This part moved to variable declaration, duplicates was removed. No need macros for one code usage